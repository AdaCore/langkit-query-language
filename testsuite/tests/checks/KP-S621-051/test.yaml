driver: 'checker'
rule_name: KP_S621_051
project: 'prj.gpr'
