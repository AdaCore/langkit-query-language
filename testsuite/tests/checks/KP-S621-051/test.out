p.adb:10:20: rule violation: possible occurrence of KP S621-051
10 |    return A_Inner (Param) = Typ (Param);  --  FLAG
   |                    ^^^^^

